crates/node-bindings/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+authors = ["Devon Govett <devongovett@gmail.com>"]
+name = "parcel-node-bindings"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+napi = {version =  "2.12.6", features = ["serde-json"]}
+napi-derive = "2.12.5"
+parcel-js-swc-core = { path = "../../packages/transformers/js/core" }
+parcel-resolver = { path = "../../packages/utils/node-resolver-rs" }
+dashmap = "5.4.0"
+xxhash-rust = { version = "0.8.2", features = ["xxh3"] }
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+parcel-dev-dep-resolver = { path = "../../packages/utils/dev-dep-resolver" }
+oxipng = "6.0.0"
```

Comment:
I recently bumped `oxipng` to `8.0.0` to resolve a compilation issue on one of the architectures - have you rebased recently on the latest `v2` changes?